client/my-sites/stats/hooks/use-notice-visibility-query.ts

```diff
@@ -7,6 +7,7 @@ const DEFAULT_SERVER_NOTICES_VISIBILITY = {
 	traffic_page_settings: false,
 	do_you_love_jetpack_stats: false,
 	commercial_site_upgrade: false,
+	focus_jetpack_purchase: false,
```

Comment:
When you say 'focus' jetpack purchase, did you mean 'force'? Just nitpick here, would it be better if we name it something like `force_jetpack_stats_purchase` or `force_stats_purchase` ?

Reply:
Yes, you are right. I wanted to avoid using `force` because users can opt out of the flow, and it also sounds much more serious. I'd like to think that we are bringing the purchase flow into their focus :)